package.json

```diff
@@ -4,7 +4,9 @@
   "description": "Working space for the ARGO Data Dictionary collaboration",
   "main": "index.js",
   "scripts": {
-    "compile": "node compile.js"
+    "compile": "node compile.js",
+    "safe_compile": "jest && node compile.js",
```

Comment:
I think we should always run tests before compile, if someone really needs to skip it they can run the command manually using node

Reply:
this is a fair point. I will do this and update the readme accordingly.

Reply:
https://github.com/icgc-argo/argo-dictionary/pull/45/commits/7471c5bc8ebab986ba526477542298906fa35040